src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -531,9 +531,11 @@ code-inner {
     text-decoration: line-through;
 }
 
-.documentation {
-    display: none;
-}
 mark {
     padding: 0;
 }
+
+/* Top right dropdown menu item*/
+.dropdown-item .show-document {
+    color: #212529;
```

Comment:
Why do we need to change the color

Reply:
Label was showing as Link in Blue color since this is tagged as <a>. Changed it to keep it consistent with other menu items.

Reply:
There is a `.text-dark` class? Would it work?

Reply:
Thanks. That's helpful. I will update the code.